app/models/manageiq/providers/infra_manager.rb

```diff
@@ -24,7 +24,7 @@ class << model_name
     #     :service
     #        :read_timeout: 1.hour
     #
-    cache_with_timeout(:ems_config, 2.minutes) { VMDB::Config.new("vmdb").config[:ems] || {} }
+    cache_with_timeout(:ems_config, 2.minutes) { ::Settings.ems }
```

Comment:
Innnnnteresting.  So, the reasoning behind the cache_with_timeout here is that we didn't want to pound the database during the period of time where we need the config.  However, since the config is now persistent in memory, there is no need to even have the cache_with_timeout here.  In fact, we can probably investigate many of the other places we used the pattern and just get rid of it entirely.

cc @jrafanie ... you'll like this one.